applications/dashboard/controllers/class.homecontroller.php

```diff
@@ -97,6 +99,15 @@ public function fileNotFound() {
     }
 
     /**
+     * Clears the url from the user's navigation preferences.
+     */
+    private function clearNavigationPreferences() {
+        $userModel = new UserModel();
+        $uri = Gdn::request()->getRequestArguments('server')['REQUEST_URI'];
```

Comment:
I know it is the same but I think it is the intended way of using the Request object.
Otherwhise why not just `$_SERVER['REQUEST_URI'];`. Way simpler.